src/acl_types.rs

```diff
@@ -0,0 +1,115 @@
+use std::time::Duration;
+
+use serde::{self, Deserialize, Serialize};
+
+/// Information related ACL token.
+/// See https://developer.hashicorp.com/consul/docs/security/acl/tokens for more information.
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct ACLToken {
+    /// Unique ID
+    #[serde(rename = "AccessorID")]
+    pub accessor_id: String,
+    /// Secret for authenticatioIDn
+    #[serde(rename = "SecretID")]
+    pub secret_id: String,
+    /// Description
+    pub description: String,
+    /// Policies
+    #[serde(default)]
+    pub policies: Vec<ACLTokenPolicyLink>,
+    /// Token only valid in this datacenter
+    #[serde(default)]
+    pub local: bool,
+    /// creation time
+    pub create_time: String,
+    /// hash
+    pub hash: String,
+    /// create index
```

Comment:
Prefer documentation casing be consistent if possible.
```suggestion
    /// Creation time
    pub create_time: String,
    /// Hash
    pub hash: String,
    /// Create index
```